chains/evm/contracts/pools/TokenPoolV2.sol

```diff
@@ -0,0 +1,142 @@
+  // │                          CCV                                  │
+  // ================================================================
+
+  function applyCCVConfigUpdates(
+    CCVConfigArg[] calldata verifierConfigArgs
+  ) external virtual onlyOwner {
+    for (uint256 i = 0; i < verifierConfigArgs.length; ++i) {
+      uint64 remoteChainSelector = verifierConfigArgs[i].remoteChainSelector;
+      if (!isSupportedChain(remoteChainSelector)) {
+        revert NonExistentChain(remoteChainSelector);
+      }
+
+      address[] calldata outboundCCVs = verifierConfigArgs[i].outboundCCVs;
+      address[] calldata inboundCCVs = verifierConfigArgs[i].inboundCCVs;
+
+      // Validate and check for duplicates in outbound CCVs
+      _validateCCVArray(outboundCCVs);
+
+      // Validate and check for duplicates in inbound CCVs
+      _validateCCVArray(inboundCCVs);
+
+      CCVConfig memory ccvConfig = CCVConfig({outboundCCVs: outboundCCVs, inboundCCVs: inboundCCVs});
+      emit CCVConfigUpdated(remoteChainSelector, ccvConfig);
+      s_verifierConfig[remoteChainSelector] = ccvConfig;
+    }
+  }
+
+  /// @notice Returns the set of required CCVs for incoming messages from a source chain.
+  /// @param sourceChainSelector The source chain selector for incoming messages.
+  /// @dev The second param is the amount and the third param is the tokenArgs.
+  /// This implementation assumes the same set of CCVs are used for all transfers on a lane.
+  /// Implementers can override this function to define custom logic based on these params.
+  /// @return requiredCCVs Set of required CCV addresses.
+  function getRequiredInboundCCVs(
+    uint64 sourceChainSelector,
+    uint256,
+    bytes calldata
+  ) external view virtual returns (address[] memory requiredCCVs) {
+    return s_verifierConfig[sourceChainSelector].inboundCCVs;
+  }
+
+  /// @notice Returns the set of required CCVs for outgoing messages to a destination chain.
+  /// @param destChainSelector The destination chain selector for outgoing messages.
+  /// @dev The second param is the amount and the third param is the tokenArgs.
+  /// This implementation assumes the same set of CCVs are used for all transfers on a lane.
+  /// Implementers can override this function to define custom logic based on these params.
+  /// @return requiredCCVs Set of required CCV addresses.
+  function getRequiredOutboundCCVs(
+    uint64 destChainSelector,
+    uint256,
+    bytes calldata
+  ) external view virtual returns (address[] memory requiredCCVs) {
+    return s_verifierConfig[destChainSelector].outboundCCVs;
+  }
+
+  /// @notice Validates a CCV array for zero addresses and duplicates.
+  /// @param ccvs The array of CCV addresses to validate.
+  function _validateCCVArray(
+    address[] calldata ccvs
+  ) private pure {
+    if (ccvs.length == 0) {
+      revert CCVArrayCannotBeEmpty();
+    }
+
+    for (uint256 i = 0; i < ccvs.length; ++i) {
+      if (ccvs[i] == address(0)) {
```

Comment:
I thought we decided to have address(0) mean the lane default. That would make it a valid input and it would need to be handled as such in the onramp/offramps

Reply:
Okay, this makes sense as long as we make it clear.